### PR TITLE
Form Fields Doc Block - copy paste and consistency

### PR DESCRIPTION
**Pull request by brianteeman**
I noticed these copy paste and consistency errors in the doc blocs. If we have doc blocks then they should be usable and correct. (they might not be perfect but they are better than they were ;) )

### CODE REVIEW

File: libraries/cms/form/field/moduletag.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 JFormHelper::loadFieldClass('list');
 
 /**
- * Form Field class for the Joomla! CMS.
+ * Moduel Tag field.
```
Comment (Quy) [inline libraries/cms/form/field/moduletag.php]: Typo

File: administrator/components/com_modules/models/fields/modulesmodule.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
 JFormHelper::loadFieldClass('list');
 
 /**
- * ModulesModule Field class for the Joomla Framework.
+ * ModulesModule field.
```
Comment (Quy) [inline administrator/components/com_modules/models/fields/modulesmodule.php]: Separate `ModulesModule`.

**Comment by Bakual** [comment]
Merged by review as it is only affecting doc blocks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**